BlueLevel/TimerRanOutScene.gd

```diff
@@ -0,0 +1,21 @@
+extends CanvasLayer
+
+func _ready():
+	pass
+
+func _process(_delta):
+	pass
+
+func _on_restart_pressed():
+	get_tree().change_scene_to_file("res://BlueLevel/world.tscn")
+	print("Clicked")
```

Comment:
looks good

Reply:
It takes a while for the world to restart after the timer runs out, I beleive there are two different values that are set, which makes the mechanic sloppy